### PR TITLE
[pr2eus/robot-interface.l] fix: too many loop angle vector duration/sequence method

### PR DESCRIPTION
**Pull request by furushchev**

### CODE REVIEW

File: pr2eus/robot-interface.l [hunk 1]
```diff
@@ -251,6 +251,11 @@
 
   (:angle-vector-duration
    (start end &optional (scale 1) (min-time 1.0) (ctype controller-type))
+   (when (or (null start) (null end))
```
Comment (k-okada) [inline pr2eus/robot-interface.l]: when this situation happens.
https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/robot-interface.l#L304 ? do we set av = nil?

https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/robot-interface.l#L357 potentio-vector (https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/robot-interface.l#L347) returns nil?

https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/robot-interface.l#L372 avs and tms has different length

File: pr2eus/robot-interface.l [hunk 2]
```diff
@@ -352,7 +357,7 @@
          (setq tms (append tms (make-list (- (length avs) (length tms)) :initial-element (car (last tms))))))
      (prog1 ;; angle-vector-sequence returns avs
          avs
-       (while avs
+       (while (cadr avs)
```
Comment (k-okada) [inline pr2eus/robot-interface.l]: what do you wan to check ? does avs contain null list, such as `(list #f(....) nil #f(...))` ...?

**Comment by furushchev** [comment]
https://github.com/jsk-ros-pkg/jsk_robot/issues/381#issuecomment-123974071
にも書きましたが、`:angle-vector-sequence`の内部で`:angle-vector-duration`が以下のように呼ばれるのが原因だと思います。

```lisp
(setq avs (list av0 av1 av2))
(send *ri* :angle-vector-sequence avs tms)
;; this calls
;; (send *ri* :angle-vector-duration av0 av1)
;; (send *ri* :angle-vector-duration av1 av2)
;; (send *ri* :angle-vector-duration av2 nil)   ;; => error
```

File: pr2eus/robot-interface.l (hunk 2 above)
Comment (furushchev) [inline pr2eus/robot-interface.l]: This is almost equivalent to:

```lisp
(let ((i 0))
 (while (< i (1- (length avs)))
  (setq av (elt avs i))
  (incf i)
  (send *ri* :angle-vector-duration av (car avs))
))
```

**Comment by k-okada** [comment]
lgtm

**Pull request merged**